package.json

```diff
@@ -3,7 +3,7 @@
   "version": "4.8.0",
   "description": "Building blocks for LoopBack connectors",
   "engines": {
-    "node": ">=6"
+    "node": ">=8"
```

Comment:
Let's use `>=8.9`. I remember that older 8.x versions are missing certain features that are important for `async`/`await`-style of programming.